CODE_REVIEW.md

```diff
@@ -0,0 +1,26 @@
+# Code Review
+
+## My REview Items
+
+1. The models file in *libs/shared/models* can be changed, leaving the "models" forlder, but spliting the model.ts creating a file for each model and its children models, naming the file as the father model, for a better present and future management.
+
+2. The *libs/books/data-access/src/lib/+state* folder can be split into multiple folders, leaving the *state* as the father folder, and having child folders for *books* and *reading-list*, as the way it is right now, complicates searching for an specific file. The change should also allow the growth of the app without creating a mess in the folder, as more states would mean more files mixed in the same place.
+
+3. On *libs/books/feature/src/lib/book-search* I think the *formatDate(date: void | string)* method shoul be taken out into a "shared" folder, on a util, as it might be required somewhere else in the future, avoiding to create the same method every time it is required.
```

Comment:
There is a better way of dealing with this...

Reply:
I added an extra observation by the moment and decided to create a pipe for the formatDate, I'll check if I can find more  issues here

*Note: Sorry, Just checked the comments yesterday on detail and having lots of issues with internet.